components/Layout/Images/BlurhashImage.vue

```diff
@@ -54,93 +66,132 @@ export default Vue.extend({
     type: {
       type: String as () => ImageType,
       default: ImageType.Primary
+    },
+    lazy: {
+      type: Boolean,
+      default: () => {
+        return true;
+      }
     }
   },
   data() {
     return {
-      image: ''
+      image: '',
+      loading: true
     };
   },
   mounted(): void {
     const img = this.$refs.img as HTMLElement;
     // We don't pass the item itself as we already did all the tags checking in this component,
     // so doing it again in the mixin is useless.
     this.image = this.getImageUrlForElement(this.type, {
-      itemId: this.item.Id,
+      itemId: this.item.Id as string,
```

Comment:
```suggestion
      itemId: this.item.Id || '',
```

We can't guarentee that `this.item.Id` will be defined

Reply:
@Camc314 I think this might be an error in the API. Why in ``BaseItemDto`` ``.Id`` is either ``string`` or ``undefined`` but in ``BaseItemPerson`` it can be ``null`` as well?